### PR TITLE
this took me five fucking hours

### PR DESCRIPTION
**Pull request by AdrienTheMan**
-fixes being able to equip paper bags containing items
~~-gives a little feature where you can switch through like 9 cool logos when alt clicking!!!!!!!! (however i couldnt get fancy polymorphic magic so the bag & logo get painted together if you dye it)~~
-paper bags are now polymorph
-they can receive a logo or pattern by alt clicking the air with help and disarm intent
-yes the logos save
-2 extra presets, NT and syndie with their respective colors
yas
image outdated but there's a newer one lower
**these additions are necessities in these trying times**

### CODE REVIEW

File: code/_core/obj/item/clothing/hats/paperbag.dm [hunk 1]
```diff
@@ -20,22 +20,80 @@
...
+
+/obj/item/clothing/head/hat/paperbag/save_item_data(var/save_inventory = TRUE)
+	. = ..()
+	SAVEVAR("logo")
+	return .
+
+/obj/item/clothing/head/hat/paperbag/load_item_data_pre(var/mob/living/advanced/player/P,var/list/object_data)
+	. = ..()
+	LOADVAR("logo")
+	update_sprite()
```
Comment (BurgerLUA) [inline code/_core/obj/item/clothing/hats/paperbag.dm:33]: update_sprite should be called in `Finalize()` instead.

**Review by BurgerLUA — CHANGES_REQUESTED** [review]
Looks good, just needs one change.

File: code/_core/obj/item/clothing/hats/paperbag.dm (hunk 1 above)
Comment (AdrienTheMan) [reply code/_core/obj/item/clothing/hats/paperbag.dm]: is this good now i dont have a big brain

Comment (AdrienTheMan) [reply code/_core/obj/item/clothing/hats/paperbag.dm]: okay nevermind i dont even need to call anything to update or finalize the sprite, it loads it up properly without

**Comment by AdrienTheMan** [comment]
okay so the bags are now polymorph compatible, can cycle through 5 logos with help intent alt click and 1 pattern (the stripe) with disarm alt click

**Review by BurgerLUA — APPROVED** [review]
Not what I meant by the requested changes but I'll update it myself.

**Pull request merged**